plugins/Toolbox/resources/qml/ToolboxShadow.qml

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Toolbox is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+
+Rectangle
+{
+    property bool reversed: false
```

Comment:
Did you intend to do something with this? It seems to always be `false`.

Reply:
Yes. It's possible to overwrite when creating an instance of it. I think in some place I set "reversed: true" when using an instance.

Reply:
Just checked. It's overwritten for the "restart bar" shadow.